tests/integration-tests/configs/develop.yaml

```diff
@@ -7,19 +7,19 @@ test-suites:
   runtime_bake:
     test_runtime_bake.py::test_runtime_bake:  # These are currently skipped
       dimensions:
-        {%- for os, scheduler in [("alinux2", "slurm"), ("alinux2", "torque"), ("centos7", "sge")] %}
+        {%- for os, scheduler in [("alinux2", "slurm")] %}
```

Comment:
you can probably remove the for loops now. Shall we leave centos7 with slurm to test the runtime bake on that os?

Reply:
Runtime bake are currently skipped and they are no longer supported but I'll let @lukeseawalker and @yuleiwan fix them.